addons/sale/sale.py

```diff
@@ -824,7 +824,11 @@ def _sales_count(self):
 
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
-    track_service = fields.Selection([('manual', 'Manually set quantities on order')], string='Track Service', default='manual')
+    track_service = fields.Selection([('manual', 'Manually set quantities on order')], string='Track Service',
+                                     help='Manually set quantities on order: It will create invoice for manually entered quanitity and no analytic account created.\n'
+                                     'Timesheets on contract: It will create invoice based on time tracking quantity (Like one hour, 5 hours).\n'
```

Comment:
If I install only sale, I have a tooltip enumerating a selection which is actually not existing.